openapi/api.github.com/operations/actions/remove-selected-repo-from-org-secret.json

```diff
@@ -0,0 +1,64 @@
+{
+  "summary": "Remove selected repository from an organization secret",
+  "description": "Removes a repository from an organization secret when the `visibility` for repository access is set to `selected`. The visibility is set when you [Create or update an organization secret](https://developer.github.com/v3/actions/secrets/#create-or-update-an-organization-secret). You must authenticate using an access token with the `admin:org` scope to use this endpoint. GitHub Apps must have the `secrets` organization permission to use this endpoint.",
+  "operationId": "actions/remove-selected-repo-from-org-secret",
+  "tags": ["actions"],
+  "externalDocs": {
+    "description": "API method documentation",
+    "url": "https://developer.github.com/v3/actions/secrets/#remove-selected-repository-from-an-organization-secret"
+  },
+  "parameters": [
+    {
+      "name": "accept",
+      "description": "Setting to `application/vnd.github.v3+json` is recommended",
+      "in": "header",
+      "schema": {
+        "type": "string",
+        "default": "application/vnd.github.v3+json"
+      }
+    },
+    {
+      "name": "org",
+      "description": "org parameter",
+      "in": "path",
+      "required": true,
+      "schema": { "type": "string" }
+    },
+    {
+      "name": "secret_name",
+      "description": "secret_name parameter",
+      "in": "path",
+      "required": true,
+      "schema": { "type": "string" }
+    },
+    {
+      "name": "repository_id",
+      "description": "repository_id parameter",
+      "in": "path",
+      "required": true,
+      "schema": { "type": "integer" }
+    }
+  ],
+  "responses": { "418": { "description": "Response definition missing" } },
```

Comment:
responses are defined at https://developer.github.com/v3/actions/secrets/#response-when-repository-was-removed-from-the-selected-list